hack/run-all.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Run all our scripts in order.
+wd="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+$wd/clean-deps.sh &&
+$wd/install-deps.sh &&
+$wd/test-integration-headless.sh &&
+$wd/verify-dist.sh
```

Comment:
verify-dist will fail if the clean/install did anything ... so I'd add grunt build. But then I'm not sure you need verify-dist at all. I'd want grunt test as well.